frontend/src/components/workspaces/Templates/TemplatesTableLogic/TemplatesTableLogic.tsx

```diff
@@ -38,7 +46,7 @@ const TemplatesTableLogic: FC<ITemplateTableLogicProps> = ({ ...props }) => {
     error: errorInstances,
     startPolling: startPollingInstances,
   } = useOwnedInstancesQuery({
-    variables: { tenantNamespace: `tenant-${userId}` },
+    variables: { tenantNamespace: tenantNamespace },
```

Comment:
```suggestion
    variables: { tenantNamespace },
```
You could avoid repeating the name and the value in objects when it's the same. There are several repetitions of this.

Reply:
Changed it